app/views/listings/index.html.erb

```diff
@@ -1,7 +1,6 @@
-<h1>Listings#index</h1>
-<p>Find me in app/views/listings/index.html.erb</p>
+<h1>Bali Sea Tours by Phil Boss</h1>
```

Comment:
This line needs to be dynamic and not hard coded

Reply:
isnt this just the header 1 for the landing page ? so it will be ok to hard code the title for the home page

Reply:
It's just a random text . feel free to change h1 for the page